bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/adapters/ColumnDescriptorAdapter.java

```diff
@@ -170,7 +170,9 @@ public static String buildGarbageCollectionExpression(HColumnDescriptor columnDe
     if (buffer.length() != 0) {
       buffer.append(" || ");
     }
-    buffer.append(String.format("(version() > %s)", maxVersions));
+    if (maxVersions != Integer.MAX_VALUE) {
```

Comment:
nit: 
add comment: When maxVersions is set to Integer.MAX_VALUE, it means we want to keep all versions. By not providing GC expression, Bigtable keeps all versions.